src/Reflection.php

```diff
@@ -59,27 +60,26 @@ public static function reflectClass($class, $argv = false, $namespace = '')
      * If extra arguments should be passed to the dispatchable function, these
      * may be provided as an array to $argv.
      *
-     * @param string $function Function name
-     * @param  bool|array $argv Optional arguments to be used during the method call
-     * @param string $namespace Optional namespace with which to prefix the
+     * @param string|callable $function Function name
+     * @param  null|array $argv Optional arguments to be used during the method call
+     * @param null|string $namespace Optional namespace with which to prefix the
      * function name (used for the signature key). Primarily to avoid
      * collisions, also for XmlRpc namespacing
-     * @return \Laminas\Server\Reflection\ReflectionFunction
-     * @throws \Laminas\Server\Reflection\Exception\InvalidArgumentException
+     * @return ReflectionFunction
+     * @throws InvalidArgumentException|ReflectionException
      */
-    public static function reflectFunction($function, $argv = false, $namespace = '')
-    {
+    public static function reflectFunction(
+        $function,
+        ?array $argv = null,
+        ?string $namespace = null
```

Comment:
Here is slightly different situation, but also signature change - default value from empty string to `null`.
We need to track down what should be the correct value, and how the code works with `null` and empty string.

Reply:
Note added to the docs.

This is a change that follows the signature change made in `AbstractFunction`, where $namespace is optional and thus nullable.